### PR TITLE
chore: Change name to luminous and edit license agreement

### PR DESCRIPTION
**Pull request by Shinyzenith**

### CODE REVIEW

**Review by Decodetalkers — APPROVED** [review]

**Comment by Decodetalkers** [comment]
maybe in README we  need to say what interfaces are realized now? 
org.freedesktop.impl.portal.ScreenCast
and
org.freedesktop.impl.portal.Screensshot

**Comment by Shinyzenith** [comment]
Sure. Will do!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,35 @@
...
+meson build
+ninja -C build install
+```
+
+# Requirements:
+
+Build time requirements are marked with `*`.
+
+1. cargo * 
+1. gstreamer *
```
Comment (Decodetalkers) [inline README.md:27]: it is not needed. gstreamer

Comment (Shinyzenith) [reply README.md:27]: Okay!

**Review by Decodetalkers — APPROVED** [review]

**Comment by Decodetalkers** [comment]
wait, the name of portal seems need to change also?

**Comment by Decodetalkers** [comment]
emm, let me change its name, lol

**Pull request merged**